inference-engine/src/inference_engine/ie_network_reader.cpp

```diff
@@ -180,6 +180,7 @@ CNNNetwork details::ReadNetwork(const std::string& modelPath, const std::string&
 
                 // read model with weights
                 auto network = reader->read(modelStream, binStream, exts);
+                binStream.close();
```

Comment:
Could you please provide the detected error?

As I know I don't need close the ifstream ([Proof](https://stackoverflow.com/questions/748014/do-i-need-to-manually-close-an-ifstream))...

Reply:
Resource acquired to 'binStream.open(weights_path, std::ios::binary)' at line 177 may be lost
Code: RH.LEAK

Reply:
Should I delete this change?

Reply:
I think we can mark it as not a problem. Because in destructor ifstream closes the file.

Reply:
i would remove model stream.close() then as well to be unified.